src/main/java/mate/academy/myJdbc/dao/DeveloperDaoImpl.java

```diff
@@ -0,0 +1,240 @@
+
+            //add skills to developer
+            statement = connection.prepareStatement("SELECT skills.skill_id, skills.skill_branch, skills.skill_level, developers.developer_id\n" +
+                    "FROM skills\n" +
+                    "JOIN developers_skills ON (skills.skill_id = developers_skills.skill_id)\n" +
+                    "JOIN developers ON (developers.developer_id = developers_skills.developer_id)\n" +
+                    "HAVING developer_id = ?;");
+            statement.setInt(1, id);
+            rs = statement.executeQuery();
+            while (rs.next()) {
+                Skill skill = new Skill(rs.getInt("skill_id"),
+                        rs.getString("skill_branch"),
+                        rs.getString("skill_level"));
+                developer.addSkill(skill);
+            }
+
+            //add projects to developer
+            statement = connection.prepareStatement("SELECT projects.project_id, projects.project_name, projects.project_cost, projects.project_birthday, developers.developer_id\n" +
+                    "FROM projects\n" +
+                    "JOIN projects_developers ON (projects.project_id = projects_developers.project_id)\n" +
+                    "JOIN developers ON (developers.developer_id = projects_developers.developer_id)\n" +
+                    "HAVING developer_id = ?;");
+            statement.setInt(1, id);
+            rs = statement.executeQuery();
+            while (rs.next()) {
+                Project project = new Project.ProjectBuilder().setId(rs.getInt("project_id")).setBirthday(rs.getDate("project_birthday")).setName(rs.getString("project_name")).build();
+                developer.addProject(project);
+            }
+
+            return developer;
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    public void updateDeveloper(Developer developer) {
+        try {
+            PreparedStatement statement = connection.prepareStatement("UPDATE developers SET developer_name=?" +
+                    ", developer_age=?, developer_salary=? WHERE developer_id=?;");
+            statement.setString(1, developer.getName());
+            statement.setInt(2, developer.getAge());
+            statement.setInt(3, developer.getSalary());
+            statement.setInt(4, developer.getId());
+            statement.execute();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void deleteDeveloper(int id) {
+        try {
+            deleteDeveloperSkillRelations(id);
+            deleteDeveloperProjectRelations(id);
+
+            PreparedStatement statement = connection.prepareStatement("DELETE FROM developers WHERE developer_id=?;");
+            statement.setInt(1, id);
+            statement.execute();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    public int getSkillId(Skill skill) {
```

Comment:
move this to skill dao. in Developer Dao we need to have operations that relates only to Developer. 
Use this method on DeveloperService layer